src/NodaTime/Duration.cs

```diff
@@ -1190,6 +1190,25 @@ void IXmlSerializable.WriteXml([NotNull] XmlWriter writer)
         [Pure]
         internal decimal ToDecimalNanoseconds() => IsInt64Representable ? ToInt64NanosecondsUnchecked() : ((decimal)days) * NanosecondsPerDay + nanoOfDay;
 
+        /// <summary>
+        /// Returns the larger duration of the given two.
```

Comment:
I was wondering how to express that. The trouble is that "longer/shorter" sound more like absolute values. Consider two durations: -2 hours and 1 hour. We want to express that 1 hour is "more than" -2 hours. I'm not sure whether either larger or longer cut it there. Any other ideas? (If not, I'm happy to just change it to longer/shorter and maybe add a remark.)